com.liferay.blade.cli/src/com/liferay/blade/cli/CreateCommand.java

```diff
@@ -102,35 +108,24 @@ else if (!isExistingTemplate(template)) {
 		projectTemplatesArgs.setService(_options.service());
 		projectTemplatesArgs.setTemplate(template);
 
-		execute(projectTemplatesArgs, isWorkspace);
+		execute(projectTemplatesArgs);
 
 		_blade.out().println(
-			"Created the project " + projectTemplatesArgs.getName() + " using the " +
-					projectTemplatesArgs.getTemplate() + " template in " + dir);
+			"Created the project " + projectTemplatesArgs.getName() +
+			" using the " + projectTemplatesArgs.getTemplate() +
+			" template in " + dir.getAbsolutePath());
 	}
 
-	void execute(ProjectTemplatesArgs projectTemplatesArgs, boolean isWorkspace) throws Exception {
+	void execute(ProjectTemplatesArgs projectTemplatesArgs) throws Exception {
 		File dir = projectTemplatesArgs.getDestinationDir();
+		String name = projectTemplatesArgs.getName();
 
 		new ProjectTemplates(projectTemplatesArgs);
 
-		if (isWorkspace) {
```

Comment:
there were the same function code in project-templates projects , but I just found Andrea removed them three days ago. So I will double check this and add it back if needed.